### ISSUE TITLE
How can we encode a DSG trajectory has two identifiers?

### ISSUE THREAD

**Issue by davidhassell** [cf-convention/discuss#282]
Hello,

I am creating a DSG trajectory dataset for meteorological research flight paths that each have two separate identifiers:
1. The name of the route flown
2. The unique ID of each individual flight

Multiple trajectory features can have the same route name (because the same route is flown on multiple occasions), but no two features have same the ID.

The ID seems to me like the best fit for the auxiliary coordinates with`cf_role=trajectory_id`, but what about the route name? 

Can we store the route names in another auxiliary coordinate variable with standard name `region`? The [conventions](https://cfconventions.org/Data/cf-conventions/cf-conventions-1.11/cf-conventions.html#geographic-regions) say that "_When data is representative of geographic regions which can be identified by names [...]. We recommend that the names be chosen from the list of [standardized region names](https://cfconventions.org/Data/cf-standard-names/docs/standardized-region-names.html) whenever possible_", which seems OK. However, the description of the `region` standard name says the contradictory "_These strings are standardised. Values must be taken from the CF standard region list._". One of these is clearly wrong!

Any thoughts on this would be appreciated, many thanks,
David

**Comment by larsbarring** [comment]
I agree with Karl in that using `region` in that ways seem to stretch the it a bit too far. As an alternative to using only long name; how about using `trajectory_id` for  *1. The name of the route flown* assuming that it is a limited set of routes  (i.e. trajectories) that are flown many times, and then use the existing standard name`realization` for *2. The unique ID of each individual flight*. The description of `realization` reads
> Realization is used to label a dimension that can be thought of as a statistical sample, e.g., labelling members of a model ensemble.
 
That is, for each route (trajectory) you have an ensemble of individual flights. Does this make sense?

**Comment by davidhassell** [comment]
Hi @taylor13 and @larsbarring,

Many thanks for your advice.

I think it clear that a standard name of `region` is not appropriate here.

I quite liked the idea of putting `cf_role = trajectory_id` on 1. (i.e. the route names auxiliary coordinate variable) since we could attach standardised attribute values to everything. However it may not be ideal, since the [conventions say]( https://cfconventions.org/cf-conventions/cf-conventions.html#coordinates-metadata) "The variable carrying the `cf_role` attribute ... must provide a **_unique_** identifier for each feature instance." (my emphasis), and the route names are not unique over the set of flights.

So, given that what I think I'll be going for is:

```
string route(n_flights) ;
    route.long_name = "Name of route for each flight (some routes are repeated)" ;
string flight(n_flights) ;
    flight.cf_role = "trajectory_id" ;
    flight.long_name = "Unique ID of each flight" ;
```
Does that look OK?
David

**Comment by taylor13** [comment]
Not being wholly familiar with ``trajectory_id`` or ``cf_role``, I shouldn't have the last word, but it makes sense to me given your above summary.
Karl
